projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -45,7 +44,13 @@ import './editor.scss';
 
 const debug = debugFactory( 'videopress:video:edit' );
 
-const { myJetpackConnectUrl } = window.videoPressEditorState;
+// Get site type.
+const { siteType, myJetpackConnectUrl } = window.videoPressEditorState;
```

Comment:
Would it be worth to add some defaults, just in case?

```suggestion
const { siteType = '', myJetpackConnectUrl } = window?.videoPressEditorState || {};
```

Reply:
Well, that will help avoid breaking the block in case videoPressEditorState is not defined. But we need to ensure it's defined. So in some way, we'll silence the error. 😬

Reply:
commited